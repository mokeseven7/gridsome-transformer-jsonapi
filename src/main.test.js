import jsonapi from './main.js';



describe('Core Test Lib', () => {
  it('json should match json', () => {
    const jsonstring = JSON.stringify({name: "Mike"});
    const API = jsonapi();
    
    expect(API.parse(jsonstring)).toMatchSnapshot();
  });
});