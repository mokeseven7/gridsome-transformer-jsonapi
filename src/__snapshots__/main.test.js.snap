// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Core Test Lib json should match json 1`] = `
Object {
  "name": "Mike",
}
`;
